roles/infrastructure/defaults/main.yml

```diff
@@ -91,7 +92,10 @@ infra__aws_vpc_az_count:            "{{ infra.aws.vpc.az_count | default(3) }}"
 infra__aws_igw_name:                "{{ infra.aws.vpc.internet_gateway.name | default([infra__namespace, infra__aws_igw_suffix] | join('-')) }}"
 infra__aws_igw_suffix:              "{{ infra.aws.vpc.internet_gateway.suffix | default(common__igw_suffix) }}"
 infra__aws_route_table_name:        "{{ infra.aws.vpc.labels.route_table | default([infra__namespace, infra__aws_route_table_suffix] | join('-')) }}"
+infra__aws_priv_route_table_name:   "{{ infra.aws.vpc.labels.route_table | default([infra__namespace, 'priv', infra__aws_route_table_suffix] | join('-')) }}"
```

Comment:
Variable should be expanded to `private` from `priv`, as we are trying to be as clear as possible in our playbooks. (`ngw` is okay since the variable would get really long, but perhaps we should consider `gateway`?)

Again, for consistency, this needs its own variable, i.e. `infra.aws.vpc.labels.route_table_private`, and a label defaulting to "priv".

Reply:
I am introducing `infra.aws.vpc.labels.public_route_table_suffix` and `infra.aws.vpc.labels.private_route_table_suffix` and getting rid off `infra__aws_route_table_suffix`.